projects/demo/src/modules/components/editor-new/editor-new.component.ts

```diff
@@ -21,8 +21,6 @@ import {default as exampleSmilesToolTemplate} from '!!raw-loader!./examples/2/sm
 import {default as example3Html} from '!!raw-loader!./examples/3/index.html';
 import {default as example3Less} from '!!raw-loader!./examples/3/index.less';
 import {default as example3Ts} from '!!raw-loader!./examples/3/index.ts';
```

Comment:
replacing examples' files by `import(...)` will destabilize tests.

It will be done in separate PR https://github.com/Tinkoff/taiga-ui/pull/1504